drafts/draft-ietf-ippm-ioam-ipv6-options.xml

```diff
@@ -320,12 +320,23 @@
             due to the high complexity in identifying the source of the leak.
             Such a troubleshooting process might require coordination between
             multiple operators, complex configuration verification,
-            packet capture analysis, etc.</t>
+            packet capture analysis, etc. This requirement may require 
+            additional option or fields to be defined to identify the domain
+            that inserted the IOAM data,  this is not addressed
+            in this document.</t>
 
             <t hangText="C6">Compliance with <xref target="RFC8200"/>
             requires OAM data to be encapsulated instead of header/option
             insertion directly into in-flight packets using the original IPv6
             header.</t>
+
+            <t hangText="C7">The IOAM Incremental Trace Option-Type expands the
+            option length that may affect the processing of extension
+            headers and options that follow IOAM options.
+            Hence when IOAM Incremental Trace Option-Type is 
```

Comment:
s/Hence when IOAM Incremental Trace Option-Type is/Hence when the IOAM Incremental Trace Option-Type is/